_pages/documentation/learning_gem5/part1/building.md

```diff
@@ -80,20 +80,6 @@ sudo apt install build-essential git m4 scons zlib1g zlib1g-dev libprotobuf-dev
         sudo apt install libprotobuf-dev python-protobuf protobuf-compiler libgoogle-perftools-dev
         ```
 
-6. [Boost](https://www.boost.org/)
```

Comment:
Let's leave one of these and say that it is an optional dependency if you want to use the SystemC implememention.

Reply:
Added.